tsl/src/continuous_aggs/invalidation.c

```diff
@@ -302,53 +298,46 @@ invalidation_hyper_log_add_entry(int32 hyper_id, int64 start, int64 end)
 /*
  * Invalidate one or more continuous aggregates.
  *
- * Add an invalidation in the given range. The invalidation is added either to
- * the hypertable invalidation log or the continuous aggregate invalidation
- * log depending on the type of the given hypertable. If the hypertable is a
- * "raw" hypertable (i.e., one that has one or more continous aggregates), the
- * entry is added to the hypertable invalidation log and will invalidate all
- * the associated continuous aggregates. If the hypertable is instead an
- * materialized hypertable, the entry is added to the cagg invalidation log
- * and only invalidates the continuous aggregate owning that materialized
- * hypertable.
+ * Add an invalidation in the given range to the continuous aggregate invalidation
+ * log.
+ *
+ * The entry is added to the hypertable invalidation log and will invalidate all
+ * the associated continuous aggregates.
  */
 void
-invalidation_add_entry(const Hypertable *ht, ContinuousAggHypertableStatus caggstatus,
-					   int32 entry_id, int64 start, int64 end)
+invalidate_continuous_agg(const Hypertable *ht, int64 start, int64 end)
 {
 	Assert(ht != NULL);
 
 	if (hypertable_is_distributed(ht))
 	{
-		remote_invalidation_log_add_entry(ht, caggstatus, entry_id, start, end);
+		ContinuousAggHypertableStatus caggstatus = ts_continuous_agg_hypertable_status(ht->fd.id);
+		remote_invalidation_log_add_entry(ht, caggstatus, start, end);
 	}
 	else
-	{
-		ContinuousAggHypertableStatus caggstatus = ts_continuous_agg_hypertable_status(ht->fd.id);
-
-		switch (caggstatus)
-		{
-			case HypertableIsMaterialization:
-				invalidation_cagg_log_add_entry(ht->fd.id, start, end);
-				break;
-
-			case HypertableIsRawTable:
-				invalidation_hyper_log_add_entry(ht->fd.id, start, end);
-				break;
+		invalidation_cagg_log_add_entry(ht->fd.id, start, end);
+}
 
-			case HypertableIsMaterializationAndRaw:
-				break;
+/*
+ * Invalidate one or more materialized hypertables.
+ *
+ * Add an invalidation in the given range to the hypertable invalidation log.
+ *
+ * The entry is added to the cagg invalidation log and only invalidates the
+ * continuous aggregate owning that materialized hypertable.
+ */
+void
+invalidate_materialized_ht(const Hypertable *ht, int64 start, int64 end)
```

Comment:
This does not invalidation a materialization hypertable at all. This is only about the underlying hypertable invalidation log. Again the comment talks both about "cagg invalidation log" and "hypertable invalidation log" which is wrong.